extensions/WEBGL_shader_pixel_local_storage/extension.xml

```diff
@@ -135,44 +129,8 @@ interface WEBGL_shader_pixel_local_storage {
   <newfun>
     <function name="framebufferPixelLocalClearValuefvWEBGL" type="undefined">
       <param name="plane" type="GLint"/>
-      <param name="value" type="sequence&lt;GLfloat&gt;"/>
-      Sets the floating point clear value for the given plane.
-      <div class="note">
-        If <code>value</code> has less than <code>4</code> elements, generates an
-        <code>INVALID_OPERATION</code> error.
-      </div>
-    </function>
-  </newfun>
-
-  <newfun>
-    <function name="framebufferPixelLocalClearValueivWEBGL" type="undefined">
-      <param name="plane" type="GLint"/>
-      <param name="value" type="sequence&lt;GLint&gt;"/>
-      Sets the integer clear value for the given plane.
-      <div class="note">
-        If <code>value</code> has less than <code>4</code> elements, generates an
-        <code>INVALID_OPERATION</code> error.
-      </div>
-    </function>
-  </newfun>
-
-  <newfun>
-    <function name="framebufferPixelLocalClearValueuivWEBGL" type="undefined">
-      <param name="plane" type="GLint"/>
-      <param name="value" type="sequence&lt;GLuint&gt;"/>
-      Sets the unsigned integer clear value for the given plane.
-      <div class="note">
-        If <code>value</code> has less than <code>4</code> elements, generates an
-        <code>INVALID_OPERATION</code> error.
-      </div>
-    </function>
-  </newfun>
-
-  <newfun>
-    <function name="framebufferPixelLocalClearValuefvWEBGL" type="undefined">
-      <param name="plane" type="GLint"/>
-      <param name="value" type="Float32Array"/>
-      <param name="src_offset" type="GLuint"/>
+      <param name="value" type="Float32List"/>
+      <param name="src_offset" type="GLuint" default="0"/>
```

Comment:
Looks like I mixed up styles for `src_offset` and `srcOffset`. Is there a preferred convention?

Also, love the `default` property!

Reply:
Could you change this to `srcOffset` in your follow-up to match the WebGL 2.0 spec? Thanks.